update installation page (#115)
JuliaImages/juliaimages.github.io: docs/src/install.md

# Getting started

Most users probably want to start with the `Images.jl` package, which bundles
much (but not all) of the functionality in JuliaImages.

## Installation

Images (and possibly some additional packages) may be all you need to manipulate images programmatically.
You can install `Images.jl` via the [package manager](https://docs.julialang.org/en/v1/stdlib/Pkg/),

```julia
(v1.0) pkg> add Images
```

!!! note
    People in some regions such as China might fail to install/precompile `Images` due to poor network
    status. Using proxy/VPN that has stable connection to Amazon S3 and Github can solve this issue.

## Loading your first image

> If this is your first time working with images in julia, it's likely that you'll need to install some

Author: FileIO no longer prompts to give installation suggestions, so I add this part at the beginning of this section.

  know what it is, then it is likely that you don't need this package.

These backends aren't exclusive to each other, so if you're a macOS user, you can install all these
backends. And in most cases, you don't need to directly interact with these backends, instead, we
use the [`FileIO.jl`](https://github.com/JuliaIO/FileIO.jl) frontend. If you've installed multiple
backends then `FileIO` will choose the most appropriate backend acoording to your file format.
For example, if available `ImageIO` is used to load png files.

Adding these gives you a basic image IO setup:

```julia
(v1.0) pkg> add FileIO ImageMagick ImageIO
```

and to load an image, you can use

```@example
using FileIO
using ImageShow # hide
img = load(joinpath("assets", "installation", "mandrill.tiff"))
```

When testing ideas or just following along with the documentation, it can be useful to have some
images to work with. The [TestImages.jl](https://github.com/JuliaImages/TestImages.jl) package
bundles several "standard" images for you.

```julia
(v1.0) pkg> add TestImages
```

To load one of the images from this package, say

> ```julia

Author: use `julia` to not display the same image twice.

  needs.
* [ImageIO.jl](https://github.com/JuliaIO/ImageIO.jl) is a new image IO backend (requires julia >=v"1.3")
  that provides an optimized performance for PNG files. Check benchmark
  [here](https://github.com/JuliaIO/PNGFiles.jl/issues/1)
* [OMETIFF.jl](https://github.com/tlnagy/OMETIFF.jl) supports
  [OME-TIFF](https://docs.openmicroscopy.org/ome-model/6.0.0/index.html#ome-tiff) files. If you don't
  know what it is, then it is likely that you don't need this package.

These backends aren't exclusive to each other, so if you're a macOS user, you can install all these
backends. And in most cases, you don't need to directly interact with these backends, instead, we
use the [`FileIO.jl`](https://github.com/JuliaIO/FileIO.jl) frontend. If you've installed multiple
backends then `FileIO` will choose the most appropriate backend acoording to your file format.
For example, if available `ImageIO` is used to load png files.

Adding these gives you a basic image IO setup:

```julia
(v1.0) pkg> add FileIO ImageMagick ImageIO
```

and to load an image, you can use

```@example
using FileIO
using ImageShow # hide

> img = load(joinpath("assets", "installation", "mandrill.tiff"))

Reviewer: From which directory should this be run? It's useful as an internal test but I am less sure it's useful to users.

Author: The new commit hides the real image path, instead, it shows an apparently invalid path to tell users to specify their own ones.

```julia
using FileIO
using ImageShow # hide
# specify the path to your local image file
img_path = "/path/to/image.png"
img_path = joinpath("assets", "installation", "mandrill.tiff") # hide
img = load(img_path)
```

![IJulia](assets/ijulia.png)

> 

Author: Not sure if removing this part is wanted.

![IJulia](assets/ijulia.png)

Currently there're five julia packages can be used to display an image:

> * [`ImageShow`](https://github.com/JuliaImages/ImageShow.jl) is used to support image display in Juno and IJulia. This is automatically used when you use `Images`.

Reviewer: ```suggestion
* [`ImageShow`](https://github.com/JuliaImages/ImageShow.jl) is used to support image display in Juno and IJulia. This happens automatically if you are `using Images`.
```
